app/Repositories/PhotoRepository.php

```diff
@@ -42,10 +42,17 @@ public function __construct(AWS $aws, Registrar $registrar)
      */
     public function create(array $data, $signupId)
     {
+        // Include signup_id when we create the Event
+        $data['signup_id'] = $signupId;
```

Comment:
@sbsmith86 It gets passed to `Event::create` with all the other data, since we now have a `signup_id` column on `events`